crates/primitives/starknet/src/crypto/commitment/mod.rs

```diff
@@ -202,8 +202,7 @@ where
     let nonce = FieldElement::from_bytes_be(&nonce.into()).unwrap();
     let version = FieldElement::from_byte_slice_be(&version.to_be_bytes()).unwrap();
     let tx_prefix = FieldElement::from_byte_slice_be(tx_prefix).unwrap();
-    // TODO: make it configurable
-    // FIXME: https://github.com/keep-starknet-strange/madara/issues/364
+
```

Comment:
take the chain id as argument so you don't have it hardcoded here

Reply:
Then we might have to add `chain_id`  as input parameter here as well
https://github.com/keep-starknet-strange/madara/blob/151dffb9bb120e45163bf7ff3c5e366ce4b5f78a/crates/primitives/starknet/src/transaction/types.rs#L340